lib/api/controllers/securityController.js

```diff
@@ -791,18 +792,12 @@ class SecurityController extends NativeController {
    * @returns {Promise<Object>}
    */
   async createRestrictedUser(request) {
-    const content = request.getBodyObject('content', {});
-    const humanReadableId = request.getString('kuid', 'human') !== 'uuid';
-
-    if (has(content, 'profileIds')) {
-      throw kerror.get('api', 'assert', 'forbidden_argument', 'body.content.profileIds');
-    }
-
-    return this._persistUser(
-      request,
-      global.kuzzle.config.security.restrictedProfileIds,
-      content,
-      { humanReadableId });
+    const under_request = new KuzzleRequest({
```

Comment:
You need to keep the whole original request context, you can look on how it's done for the JSON Query endpoint here: https://github.com/kuzzleio/kuzzle/blob/master/lib/core/network/router.js#L113